code/_globalvars/lists/flavor_misc.dm

```diff
@@ -38,8 +38,11 @@ GLOBAL_LIST_EMPTY(wings_list)
 GLOBAL_LIST_EMPTY(wings_open_list)
 GLOBAL_LIST_EMPTY(r_wings_list)
 GLOBAL_LIST_EMPTY(moth_wings_list)
+GLOBAL_LIST_EMPTY(moth_wings_sanitized_list)//this lacks the blacklisted wings such as burned, clockwork and angel
```

Comment:
name these roundstart_choices_list or something, sanitized doesn't really make sense

Reply:
That's the name I went with in the cocoon PR but then I realized this also goes into random mob creation and admin mob creation. But if you say so, I'll change it to roundstart_list

Reply:
Done